agents/postgres/pgstatmonitor/models.go

```diff
@@ -49,8 +49,14 @@ type pgStatMonitorSettings struct {
 //
 //reform:pg_stat_monitor_settings
 type pgStatMonitorSettingsTextValue struct {
-	Name  string `reform:"name"`
-	Value string `reform:"value"`
+	Name         string  `reform:"name"`
+	Value        string  `reform:"value"`
+	DefaultValue string  `reform:"default_value"`
+	Description  string  `reform:"description"`
+	Minimum      *int64  `reform:"minimum"`
+	Maximum      *int64  `reform:"maximum"`
+	Options      *string `reform:"options"`
+	Restart      string  `reform:"restart"`
```

Comment:
let's keep only fields that we use in case PGSM team changes structure later we will have more PMM clients supported latest PGSM

Reply:
Done.